README.md

```diff
@@ -9,6 +9,10 @@ A template repo to use for new repositories.
 [![Documentation Status](https://readthedocs.org/projects/connPFM/badge/?version=latest)](http://connPFM.readthedocs.io/en/latest/?badge=latest)
 [![Codecov](https://codecov.io/gh/SPiN-Lab/connPFM/branch/main/graph/badge.svg)](https://codecov.io/gh/SPiN-Lab/connPFM)
 
+## Installation
```

Comment:
Maybe these lines should be code lines so they can be copy pasted into the terminal.